src/vs/workbench/contrib/terminalContrib/suggest/browser/terminalCompletionItem.ts

```diff
@@ -17,6 +17,8 @@ export enum TerminalCompletionItemKind {
 	Option = 5,
 	OptionValue = 6,
 	Flag = 7,
+	SymbolicLinkFile = 8,
+	SymbolicLinkFolder = 9,
 	// Kinds only for core
 	InlineSuggestion = 100,
 	InlineSuggestionAlwaysOnTop = 101,
```

Comment:
The ones above that comment all live in the API so should also be duplicated in the `TerminalCompletionItemKind` proposal.